recipes/master.rb

```diff
@@ -77,12 +76,11 @@
       action :modify
     end
 
-    include_recipe 'krb5_utils'
```

Comment:
Are the krb5 LWRPs idempotent?  Minor concern here that we are moving keytab generation from where it's likely run every chef invocation, to somewhere users may not run more than once, if ever.